src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,25 @@
+const CHECK_STATUS = 'bookstore/CHECK_STATUS';
+
+export const addBookItem = (payload) => ({
+  type: CHECK_STATUS,
+  payload,
+});
+
+const initialState = {
+  categories: 'Under construction',
+};
+
+const categories = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case CHECK_STATUS:
+      return {
+        ...state,
+        categories: payload,
+      };
+
+    default:
+      return state;
+  }
+};
+
+export default categories;
```

Comment:
- [ ] Similarly, the generation of _action_ _types_ and _creators_ should be done by using the `createSlice` from `@reduxjs/toolkit` package. Also, let's ensure to:
  * Exports by default `categoriesSlice.reducer`
  * Exports by `checkStatus` as actions. 
  
  Example is as below:
```javascript
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```